export const CATCH_ERROR = 'CATCH_ERROR'
export const RESOLVE_ERROR = 'RESOLVE_ERROR'
